backend/app/mcp_server/schema_extractor.py

```
@@ -0,0 +1,480 @@
+        field_schema = _pydantic_model_to_schema(
+            annotation, max_depth, current_depth + 1
+        )
+
+    return field_schema
+
+
+# ============== Docstring Generation ==============
+
+
+def generate_tool_docstring(
+    name: str,
```

Ruff (0.14.14)

[warning] Unused function argument: `name`

(ARG001)